test/automated-suite.js

```diff
@@ -72,20 +72,14 @@ describe('Test Propel', function() {
 
       globalDriverReference = browserInfo.getSeleniumDriver();
 
-      let initialisePromise;
-
-      switch (browserInfo.seleniumBrowserId) {
-        case 'firefox':
-          // H/T to web-push for this trick to get permissions accepted / denied
-          initialisePromise = globalDriverReference.executeScript(() => {
-            /* global window, Components */
-            window.netscape.security.PrivilegeManager.enablePrivilege('UniversalXPConnect');
-            Components.utils.import('resource://gre/modules/Services.jsm');
-          });
-          break;
-        default:
-          initialisePromise = Promise.resolve();
-          break;
+      let initialisePromise = Promise.resolve();
+      if (browserInfo.seleniumBrowserId === 'firefox') {
+        // H/T to web-push for this trick to get permissions accepted / denied
+        initialisePromise = globalDriverReference.executeScript(() => {
+          /* global window, Components */
+          window.netscape.security.PrivilegeManager.enablePrivilege('UniversalXPConnect');
```

Comment:
Assume this is the line you were referring to that didn't have any external docs available around it. Can we link up to web-push's repo given we're referring to it a line or two above? (in case folks wonder what that project is)